### PR TITLE
Correcting type on word error

### LINKED ISSUE #40533: Typo in Quality Assurance Project - Sudoku Solver
[https://www.freecodecamp.org/learn/quality-assurance/quality-assurance-projects/sudoku-solver](url)

This is how one of the instructions look like:
_If the puzzle submitted to /api/check is greater or less than 81 characters, the returned value will be { **error\&#<span></span>58;** 'Expected puzzle to be 81 characters long' }_

This is how it should be:
_If the puzzle submitted to /api/check is greater or less than 81 characters, the returned value will be { **error:** 'Expected puzzle to be 81 characters long' }_

image

### PR DESCRIPTION
**Pull request by Durbatuluk1701**
Bug Fix for #40533

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40533 

Simple Typo that has been corrected, no functionality or test-case changes.

### CODE REVIEW

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳

Please note that our workflow requires at least one additional approval before this can be merged. 🙂Additionally, we are currently in a code freeze and are not merging learn PRs while that is in affect.

**Review by RandellDawson — APPROVED** [review]
LGTM

**Comment by RandellDawson** [comment]
Congrats @Durbatuluk1701 on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**
